internal/receiver/awscontainerinsightreceiver/host/ebsvolume.go

```diff
@@ -66,13 +66,13 @@ func (e *EbsVolume) Shutdown() {
 	close(e.shutdownC)
 }
 
-func (e *EbsVolume) refresh() {
+func (e *EbsVolume) refresh(region string) {
 	if e.instanceId == "" {
 		return
 	}
 
 	e.logger.Info("Fetch ebs volumes from ec2 api")
-	sess, err := session.NewSession(&aws.Config{})
+	sess, err := session.NewSession(&aws.Config{Region: aws.String(region)})
```

Comment:
Create the config conditionally?

* if region is empty, use the default one
* otherwise, use `&aws.Config{Region: aws.String(region)}`